EnhancedMediaPlayer/MediaPlayerViewController.swift

```diff
@@ -55,11 +71,21 @@ public class MediaPlayerViewController: UIHostingController<MediaPlayerView> {
     }
     
     private func rewind() {
```

Comment:
you're kind of repeating yourself here, as again, this is almost duplicated code.

do:

```
private func seek(forward: Bool) {
    let current = viewModel.player.currentTime().seconds
    let target: Double = forward ? current + Constants.seekFactor : current - Constants.seekFactor
    let result: CMTime =  .init(seconds: target, preferredTimescale: Constants.timeScale  )
    viewModel.player.seek(to: result)
}
```

and then

```swift
private func rewind() {
   seek(forward: false)
}

private func forward() {
   seek(forward: true)
}
```